regression/contracts/assigns_enforce_structs_07/test.desc

```diff
@@ -3,13 +3,18 @@ main.c
 --enforce-contract f1 --enforce-contract f2 _ --malloc-may-fail --malloc-fail-null --pointer-check
 ^EXIT=10$
 ^SIGNAL=0$
-^\[f1.\d+\] line \d+ Check that p->buf\[\(.*\)0\] is assignable: SUCCESS$
+^\[f1.\d+\] line \d+ Check that p->buf\[\(.*\)0\] is assignable: (SUCCESS|FAILURE)$
 ^\[f1.pointer\_dereference.\d+\] line \d+ dereference failure: pointer invalid in p->buf\[\(.*\)0\]: FAILURE$
 ^\[f1.pointer\_dereference.\d+\] line \d+ dereference failure: pointer NULL in p->buf: FAILURE$
-^\[f2.\d+\] line \d+ Check that pp->p->buf\[\(.*\)0\] is assignable: SUCCESS$
+^\[f2.\d+\] line \d+ Check that pp->p->buf\[\(.*\)0\] is assignable: (SUCCESS|FAILURE)$
```

Comment:
As above: we don't care?

Reply:
same as above, the inclusion check is meaningless if the target validity assertion does not hold, so it can have either status, and we were missing a check on the target validity status.